### PR TITLE
Fix broken modal styling from 1.16.8

### PR DESCRIPTION
**Pull request by mpretty-cyro**
- Fixed an issue where a bunch of dialog backgrounds were missing
- Fixed an issue where permission dialogs were illegible

### CODE REVIEW

**Comment by mpretty-cyro** [comment]
Looks like there are still a few other modals with issues so changing this to a draft PR until I've had the chance to go through them

**Comment by fracting** [comment]
Also the download modal.

download

**Comment by fracting** [comment]
Setting dialog of "Enable Beta Call feature"

enable call beta

File: app/src/main/res/values/themes.xml [hunk 1]
```diff
@@ -248,7 +248,7 @@
     </style>
 
     <style name="Theme.TextSecure.Dialog.MediaSendProgress" parent="@android:style/Theme.Dialog">
-        <item name="android:colorBackground">?attr/dialog_background_color</item>
+        <item name="android:background">@drawable/default_dialog_background</item>
```
Comment (bemusementpark) [inline app/src/main/res/values/themes.xml:251]: this sets background on every `View` so looks highly suspect.

There is only one usage though...

```
dialog = new AlertDialog.Builder(new ContextThemeWrapper(requireContext(), R.style.Theme_TextSecure_Dialog_MediaSendProgress))
                                  .setView(R.layout.progress_dialog)
                                  .setCancelable(false)
                                  .create();
```

and  the views in that layout are:

```
<RelativeLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:layout_width="wrap_content"
    android:layout_height="wrap_content"
    android:padding="@dimen/medium_spacing">

    <com.github.ybq.android.spinkit.SpinKitView
        style="@style/SpinKitView.Large.ThreeBounce"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_centerInParent="true"
        app:SpinKit_Color="?android:textColorPrimary" />

</RelativeLayout>
```

so this just sets both of them to have `default_dialog_background` as a background. and they have the same dimensions, so you won't see the background of the `RelativeLayout`

I think it'd be more correct to use `android:windowBackground` but isn't hugely important atm.

File: app/src/main/java/org/thoughtcrime/securesms/home/UserDetailsBottomSheet.kt [hunk 2]
```diff
@@ -117,8 +117,7 @@ class UserDetailsBottomSheet: BottomSheetDialogFragment() {
     override fun onStart() {
         super.onStart()
         val window = dialog?.window ?: return
-        val isLightMode = UiModeUtilities.isDayUiMode(requireContext())
-        window.setDimAmount(if (isLightMode) 0.1f else 0.75f)
+        window.setDimAmount(0.6f)
```
Comment (bemusementpark) [inline app/src/main/java/org/thoughtcrime/securesms/home/UserDetailsBottomSheet.kt:120]: cool, these should look better now!

**Review by bemusementpark — APPROVED** [review]

File: app/src/main/res/values/themes.xml (hunk 1 above)
Comment (mpretty-cyro) [reply app/src/main/res/values/themes.xml:251]: Yea this is the last one I need to properly test - am having a hard time finding it via the actual UI 😂 

If I find it and can confirm it wasn't actually broken then I'll change it back to what you had it as

Comment (bemusementpark) [reply app/src/main/res/values/themes.xml:251]: `MediaSendFragment#sendButton` ...

Screen Shot 2023-06-06 at 5 46 43 pm

but it only shows if it takes longer than 250ms...

`Util.runOnMainDelayed(progressTimer, 250);`

Comment (mpretty-cyro) [reply app/src/main/res/values/themes.xml:251]: Looks like it's actually busted with either approach (white on white) so have reverted this back to your change and updated the progress to be the accent colour:
Screenshot 2023-06-06 at 6 28 22 pm

**Comment by fracting** [comment]
Confirming the modal issue fixed in Release 1.1.6.9 for me.

**Pull request merged**